src/http.c

```diff
@@ -925,17 +983,71 @@ http_file_rewind(struct http_file *file)
 	file->offset = 0;
 }
 
-void
+size_t
+http_populate_cookies(struct http_request *req)
+{
+	int						 i, v;
+	size_t					 nlen;
+    struct http_cookie		*ck;
+    char					*c, *header, *value;
+    char					*cookies[HTTP_MAX_COOKIES];
+    const char				*semicol = ";";
+    const char				*comma = ",";
+    const char				*delim = NULL;
+
+    if (!http_request_header(req, "cookie", &c))
+	   return 0;
+
+    header = kore_strdup(c);
+    if (strstr(header, semicol) != NULL) {
+    	delim = semicol;
```

Comment:
There are several RFCs on subject and they do not restrict it explicitly. 
RFC 2109 was obsoleted by RFC 2965, which was obsoleted by RFC 6265.
A lot of browsers actually did (or still do), see https://bugzilla.mozilla.org/show_bug.cgi?id=370639.
The convention is ";" however "," is explicitly mentioned in RFC 2109. Most servers and clients support  RFC 2109 AFAIK.